descriptors/ModuleDescriptor-template.json

```diff
@@ -56,15 +56,15 @@
     },
     {
       "id": "locations",
-      "version": "2.1 3.0"
+      "version": "3.0"
```

Comment:
Why doesn't gobi need 2.1 anymore?

Reply:
Initially both versions were left just incase old one was present, It is now safe to assume 3.0 will be present always as mod-inventory-storage had more then 7-8 releases after the upgrade